harry-core/src/harry/reconciler/Reconciler.java

```diff
@@ -189,26 +199,27 @@ protected void afterLts(long lts, long pd)
                             }
 
                             partitionState.write(cd,
-                                                 descriptorSelector.vds(pd, cd, lts, opId, op.opType, schema),
+                                                 writeOp.vds(),
                                                  lts,
-                                                 op.opType == OpSelectors.OperationKind.INSERT || op.opType == OpSelectors.OperationKind.INSERT_WITH_STATICS);
+                                                 op.opKind() == OpSelectors.OperationKind.INSERT || op.opKind() == OpSelectors.OperationKind.INSERT_WITH_STATICS);
```

Comment:
nit: Might work to just name this `opKind()` -> `kind()`, so it reads `op.kind()`